scripts/test-runner.sh

```diff
@@ -29,6 +29,8 @@ if [[ ! " ${SUPPORTED_ENGINES[*]} " =~ " ${ENGINE} " ]]; then
 	helpFunction
 fi
 
+echo "$MODE://localhost:$WEB3_SYSTEM_TEST_PORT"
+echo $BACKEND
```

Comment:
👍 

Try to add further descriptions here as well. So when we check on console we know what was echoed.